KachnaOnline.App/Controllers/EventsController.cs

```diff
@@ -182,23 +346,23 @@ public async Task<ActionResult> ModifyEvent(int eventId, BaseEventDto baseEvent)
         /// Deletes an event with the given ID.
         /// </summary>
         /// <param name="eventId">ID of the event to delete.</param>
-        /// <response code="204">The event was deleted.</response>
+        /// <returns>A list of <see cref="StateDto"/> of linked states at the time of deletion.</returns>
+        /// <response code="200">The event was deleted. The list of linked states at the time of deletion.</response>
         /// <response code="404">The event does not exist.</response>
         /// <response code="409">A past event cannot be removed.</response>
         [HttpDelete("{eventId}")]
-        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(typeof(IEnumerable<StateDto>), StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status409Conflict)]
-        public async Task<ActionResult> RemoveEvent(int eventId)
+        public async Task<ActionResult<IEnumerable<StateDto>>> RemoveEvent(int eventId)
```

Comment:
Same as above.

Reply:
Yes, the controller documentation and its method decorators are still work-in-progress a bit. I am working on updating these right now.

Reply:
Done.